app/src/main/res/values/strings.xml

```diff
@@ -131,7 +131,7 @@
     <string name="quest_buildingLevels_title_buildingPart">"How many levels does this building part have?"</string>
     <string name="quest_roofShape_title">"What basic shape does this building’s roof have?"</string>
     <string name="quest_roofShape_select_one">"Select one:"</string>
-    <string name="quest_roofShape_show_more">"Show more…"</string>
+    <string name="quest_roofShape_show_more">"Show more…"</string><!--it is used in generic list, not only for roof shape but rename would break translations-->
```

Comment:
Don't the translations get uploaded from SC to the translation system, therefore couldn't you do a sed on all the strings files to replace quest_roofShape_show_more with quest_show_more and mean no translation is required?